omod/src/main/java/org/openmrs/module/providermanagement/fragment/controller/ProviderEditFragmentController.java

```diff
@@ -312,6 +313,43 @@ else if (ProviderManagementUtils.clearTimeComponent(date).after(new Date())) {
 
     }
 
+    public FragmentActionResult editSupervisor(@RequestParam(value = "supervisee", required = true) Person supervisee,
+                                               @RequestParam(value = "supervisor", required = true) Person supervisor,
+                                               @RequestParam(value = "relationship", required = false) Relationship relationship,
+                                               @RequestParam(value = "date", required = false) Date date,
+                                               @SpringBean("providerManagementService") ProviderManagementService providerManagementService) {
+
+        if (supervisee == null) {
```

Comment:
Thanks @mogoodrich . I will remove this error message and leave required=true for the supervisee.